content.md

```diff
@@ -95,4 +95,53 @@
 
 ## 3. Registering a Member
 
+### 3.1 Start from map view(2.6.) and tap menu-icon at top-left of screen
+![screenshot](./images/map.jpg)
+<div class="pagebreak"></div>
+
+### 3.2 Select *"Member Registration"* from dropdown menu
+![screenshot](./images/menu.jpg)
+<div class="pagebreak"></div>
+
+### 3.3 Select *"+Member Registration"* with the big plus sign from the *"Member Registration"* sub-menu
+![screenshot](./images/3/5.jpg)
+<div class="pagebreak"></div>
+
+### 3.4 Fill in the member details
+![screenshot](./images/3/3.jpg)
+<div class="pagebreak"></div>
+
+### 3.5 If you have connectivity tap *"Save and submit"* else tap *"Save only"*
+![screenshot](./images/3/4.jpg)
+<div class="pagebreak"></div>
+
+### 3.6 Member has been added
```

Comment:
Please add any intermediate displays of save progress or success/failure messages.  (apply everywhere)